client/src/templates/Challenges/utils/index.test.js

```diff
@@ -0,0 +1,26 @@
+/* global expect */
+
+import { forumLocation } from '../../../../../config/env.json';
+
+const { getGuideUrl } = require('./');
+
+describe('index', () => {
+  describe('getGuideUrl', () => {
+    it('should use forum topic url when forumTopicId is supplied', () => {
+      const value = getGuideUrl({
+        forumTopicId: 12345,
+        title: 'a sample title'
+      });
+      expect(value).toEqual(`${forumLocation}/t/12345`);
+    });
+
+    it('should use search endpoint when no forumTopicId is supplied', () => {
+      const value = getGuideUrl({
+        title: 'a sample title'
```

Comment:
```suggestion
        title: '& a sample title?'
```
I think we should add a couple of special characters so we're sure they're encoded properly